client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -33,7 +31,8 @@ export default React.createClass( {
 		jetpack: React.PropTypes.bool,
 		compact: React.PropTypes.bool,
 		feature: React.PropTypes.oneOf( [ false, ...getValidFeatureKeys() ] ),
-		shouldDisplay: React.PropTypes.func
+		shouldDisplay: React.PropTypes.func,
+		site: React.PropTypes.object,
```

Comment:
To give you a bit more background, this is very useful when we have component tests that use the base class, vs the wrapped class.

```
export class Foo extends Component {
//...
};
export default localize( Foo );
```

And in a test we do:
```
import { Foo } from 'components/foo';
// where output from enzyme will still be correct since the default prop value is identity
```